ibis/backends/bigquery/__init__.py

```diff
@@ -438,47 +438,82 @@ def create_table(
         database: str | None = None,
         temp: bool | None = None,
         overwrite: bool = False,
+        default_collate: str | None = None,
+        partition_by: str | None = None,
+        cluster_by: Iterable[str] | None = None,
+        options: Mapping[str, Any] | None = None,
     ) -> ir.Table:
```

Comment:
I know this was already undocumented before, but since we're adding a bunch of bigquery-specific parameters here, it might be nice to add a docstring.

Reply:
Whoops, yeah I meant to add those!

Reply:
Done, documented all the things.